### PR TITLE
Added "Open in safari" activity

### PR DESCRIPTION
**Pull request by dbergman**
@ryang1428 Take a peak when you get a chance, trying to use safari activity

### CODE REVIEW

**Comment by dbergman** [comment]
Added safari image for activity and the ability to override that image.

File: Pod/Classes/URBNActivityViewController.m [hunk 1]
```diff
@@ -7,23 +7,45 @@
...
+
+@property (nonatomic, assign) BOOL canOpenInSafari;
+@property (nonatomic, strong) URBNSafariActivity *safariActivity;
+
+@end
 
 @implementation URBNActivityViewController
 
 #pragma mark - Initilization
 - (instancetype)initWithDefaultBody:(NSString *)defaultBody {
```
Comment (jgrandelli) [inline Pod/Classes/URBNActivityViewController.m]: Why is this method still around? Did you not delete it from the H?

File: URBNShareKit.podspec [hunk 2]
```diff
@@ -10,8 +10,8 @@ Pod::Spec.new do |s|
   s.platform     = :ios, '7.0'
   s.requires_arc = true
 
-  s.source_files = 'Pod/Classes'
+  s.source_files = 'Pod/Classes', 'Pod/Assets'
```
Comment (jgrandelli) [inline URBNShareKit.podspec]: Do you need to include the assets directory as a `source_file`? If so, just use a wildcard to get all sub directories in Pod/

File: Pod/Classes/URBNActivityViewController.m (hunk 1 above)
Comment (dbergman) [inline Pod/Classes/URBNActivityViewController.m]: fixed.

File: URBNShareKit.podspec (hunk 2 above)
Comment (dbergman) [inline URBNShareKit.podspec]: fixed.

File: Pod/Classes/URBNActivityViewController.m (hunk 1 above)
Comment (ryang1428) [inline Pod/Classes/URBNActivityViewController.m]: Are we going to go with just the 1 ugly init?

**Pull request merged**